core/field_textinput.js

```diff
@@ -386,9 +386,15 @@ Blockly.FieldTextInput.prototype.showEditor_ = function(_opt_e,
   this.workspace_ = this.sourceBlock_.workspace;
   var quietInput = opt_quietInput || false;
   var readOnly = opt_readOnly || false;
+  var isTouchEvent = undefined;
+  if (_opt_e instanceof PointerEvent) {
+    isTouchEvent = _opt_e.pointerType === "touch";
+  }
+
   if (!quietInput && (Blockly.utils.userAgent.MOBILE ||
                       Blockly.utils.userAgent.ANDROID ||
-                      Blockly.utils.userAgent.IPAD)) {
+                      Blockly.utils.userAgent.IPAD) ||
```

Comment:
I think the `|| isTouchEvent` should be with the other ios events inside the paranetheses, so quietInput has an effect?

Reply:
oh shoot yes something got messed up ty joey